crates/contracts/src/precompiles/tip_fee_manager.rs

```diff
@@ -99,7 +99,7 @@ sol! {
         function rebalanceSwap(address userToken, address validatorToken, uint256 amountOut, address to) external returns (uint256 amountIn);
 
         // Events
-        event Mint(address indexed sender, address indexed userToken, address indexed validatorToken, uint256 amountUserToken, uint256 amountValidatorToken, uint256 liquidity);
+        event Mint(address sender, address indexed to, address indexed userToken, address indexed validatorToken, uint256 amountValidatorToken, uint256 liquidity);
```

Comment:
Noting that we will want to flag this to front end teams to let them know the indexed field has changed. Since the previously indexed field was also `address` this could go unnoticed.